addons/kommander/1.2.0/kommander-12.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.0-11"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.2.0-beta.2"
```

Comment:
Maybe this, too but @juliangieseke might know better.

Reply:
this one not. we'll bump the appversion when doing released - nothing that needs to be done in regular PRs.